File: CS50x-2024-Pset01-C/mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

void printSpace(int height, int rowNumber);
void printHash(int rowNumber);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1);

    for (int i = 0; i < height; i++)
    {
        printSpace(height, i);
        printHash(i);
        printf("  ");
        printHash(i);
        printf("\n");
    }
    return 0;
}

void printSpace(int height, int rowNumber)
{
    for (int i = 0; i < height - 1 - rowNumber; i++)
    {
        printf(" ");
    }
}

void printHash(int rowNumber)
{
    for (int i = 0; i < rowNumber + 1; i++)
    {
        printf("#");
    }
}

File: CS50x-2024-Pset06-Python/sentimental-mario-more/mario.py
from cs50 import get_int

while True:
    try:
        height = get_int("Height: ")
        if height >= 1 and height <= 8:
            break
    except ValueError:
        pass

for i in range(height):
    print(" " * (height - 1 - i), end="")
    print("#" * (i + 1), end="")
    print("  ", end="")
    print("#" * (i + 1), end="")
    print()
